packages/cronie.rb

```diff
@@ -82,7 +90,6 @@ def self.postinstall
     puts 'To list cron tasks, execute the following:'.lightblue
     puts 'sudo crontab -l'.lightblue
     puts
-    puts 'To start the crond daemon at login, execute the following:'.lightblue
     puts "echo '# start the crond daemon' >> ~/.bashrc".lightblue
     puts "echo 'if [ -f #{CREW_PREFIX}/bin/startcrond ]; then' >> ~/.bashrc".lightblue
     puts "echo '  #{CREW_PREFIX}/bin/startcrond' >> ~/.bashrc".lightblue
```

Comment:
Add to `#{CREW_DEST_PREFIX}/etc/bash.d` instead of `~/.bashrc`?

Reply:
That's part of a wider range of changes, and I'll do those in another PR.